src/app/[locale]/incident/add/components/questions/LocationSelect.tsx

```diff
@@ -72,9 +72,14 @@ export const LocationSelect = ({ field }: LocationSelectProps) => {
                   {t('choose_location')}
                 </Button>
               ) : (
-                <LinkButton inline={true} id="location-button" type="button">
+                <Button
```

Comment:
Volgens mij bedoelde @Robbert in #229 dat deze button in een`<ButtonGroup>` Component geplaatst kan worden?
(Zie [Utrecht Storybook](https://nl-design-system.github.io/utrecht/storybook-react/?path=/docs/react-button-group--docs))